### ISSUE TITLE
I'm really sorry for bothering you.

### ISSUE THREAD

**Issue by 111678900** [ellisdg/3DUnetCNN#331]
I'm really sorry for bothering you. Your code is designed for four channels, but I only need one channel for my application. I have already changed the number of channels. Could you please let me know what other parts I need to modify?

**Comment by ellisdg** [comment]
If you haven't already, checkout BraTS example notebook for an explanation of the different arguments:
https://github.com/ellisdg/3DUnetCNN/blob/master/examples/brats2020/create_config_lowmem.ipynb

Some arguments you may consider changing:
- "desired_shape"
- "setup_label_hierarchy" should be false since you only have a mask as output based on your previous comment
- "resample" and "crop_foreground" may not be necessary depending on what your dataset.

**Comment by 111678900** [comment]
![718b1af33889dbe20441f44aa34227f](https://github.com/ellisdg/3DUnetCNN/assets/152053751/79aa7f6b-3ff8-4774-9566-e5303458e6d3)
I'm really sorry for bothering you，I'm using it to train in my own dataset.Discovering these images in the example folder after training, does it mean that the training was successful?"If it is successful, how should I run the predict.py file?

**Comment by ellisdg** [comment]
Those are 3D renderings that I made as an example of skull completion using the AutoImplant dataset.

You should have a training log file that you can look at to see how many epochs were run and if the training and validation losses decreased like you would expect.
